Sources/__SwiftOnServer_org/Documentation.docc/2024/02/08-using-swiftnio-channels/08-using-swiftnio-channels.md

```diff
@@ -1,33 +1,18 @@
----
-slug: using-swiftnio-channels
-title: Using SwiftNIO - Channels
-description: Create a TCP server using SwiftNIO and structured concurrency
-date: 2024/02/08
-tags: Swift, SwiftNIO, Networking
-author: Joannis Orlandos
-authorLink: https://x.com/JoannisOrlandos
-authorGithub: joannis
-authorAbout: Joannis is a seasoned member of the Swift Server WorkGroup, and the co-founder of Unbeatable Software B.V. If you're looking to elevate your team's capabilities or need expert guidance on Swift backend development, consider hiring him.
-cta: Get in touch with Joannis
-ctaLink: https://unbeatable.software/mentoring-and-training
-company: Unbeatable Software B.V.
-companyLink: https://unbeatable.software/
-duration: 30 minutes
----
-
 # SwiftNIO Channels
 
-In the [previous tutorial](/using-swiftnio-fundamentals), you've learned the fundamentals of SwiftNIO. You're now familiar with the concept of an `EventLoop`.
+SwiftNIO revolves around networking. In SwiftNIO, reads and writes happen using a ``Channel``.
+
+In the [previous tutorial](https://swiftonserver.com/using-swiftnio-fundamentals), you've learned the fundamentals of SwiftNIO. You're now familiar with the concept of an ``EventLoop``.
```

Comment:
[this PR](https://github.com/tayloraswift/swift-unidoc/pull/207), which i just deployed to the prod site a few minutes ago, should enable you to make this a `doc:` link again, if you also rename the source files to the desired URL path component